phase1/openstack/configure-vm-kubeadm.sh

```diff
@@ -0,0 +1,35 @@
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+
+mkdir -p /var/lib/kubelet
+
+if [[ "${k8s_kubeadm_version}" == stable ]]; then
+  cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+  deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+
+  apt-get update
+  apt-get install -y kubelet kubeadm kubectl kubernetes-cni
+elif [[ "${k8s_kubeadm_version}" == "gs://"* ]]; then
+  TMPDIR=/tmp/k8s-debs
+  mkdir $TMPDIR
+  apt-get install -y python
+  cd $TMPDIR
+  wget https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz
```

Comment:
Do we need the google-cloud-sdk for openstack environments?

Reply:
Never mind, looks like we are downloading the various deb(s) using gsutil

Reply:
yes, google-cloud-sdk is to install gsutil.
currently the .deb packages are released and stored in google cloud storage.